packages/api-extractor/package.json

```diff
@@ -0,0 +1,80 @@
+{
+	"name": "@discordjs/api-extractor",
+	"version": "7.38.1",
+	"description": "Analyze the exported API for a TypeScript library and generate reviews, documentation, and .d.ts rollups",
+	"private": true,
+	"keywords": [
+		"typescript",
+		"API",
+		"JSDoc",
+		"AEDoc",
+		"TSDoc",
+		"generate",
+		"documentation",
+		"declaration",
+		"dts",
+		".d.ts",
+		"rollup",
+		"bundle",
+		"compiler",
+		"alpha",
+		"beta"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/discordjs/discord.js.git",
+		"directory": "packages/api-extractor"
+	},
+	"homepage": "https://discord.js.org",
+	"bin": {
+		"api-extractor": "./bin/api-extractor"
+	},
+	"license": "MIT",
+	"scripts": {
+		"build": "tsc --noEmit && tsup && pnpm run copy-files",
+		"copy-files": "cp -r ./src/schemas ./dist/ && cp -r ./extends ./dist/",
```

Comment:
Fairly sure that only works correctly on linux and macOS.

Maybe install `cpy-cli` as a devDep and use that instead?

In the package.json for the website:
```json
"build:copy_readme": "cpy '../../packages/*/README.md' 'src/assets/readme' --rename='home-{{basename}}'",
```